WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -656,30 +630,22 @@ - (void)showBlogSelectorPrompt:(WPBlogSelectorButton*)sender
         [self showBlogSelector];
         return;
     }
-    UIAlertView *alertView = [[UIAlertView alloc] initWithTitle:NSLocalizedString(@"Change Site", @"Title of an alert prompting the user that they are about to change the blog they are posting to.")
-                                                        message:NSLocalizedString(@"Choosing a different site will lose edits to site specific content like media and categories. Are you sure?", @"And alert message warning the user they will loose blog specific edits like categories, and media if they change the blog being posted to.")
-                                                       delegate:self
-                                              cancelButtonTitle:NSLocalizedString(@"Cancel",@"")
-                                              otherButtonTitles:NSLocalizedString(@"OK",@""), nil];
-    alertView.tag = EditPostViewControllerAlertTagSwitchBlogs;
-    [alertView show];
+    UIAlertController *alertController = [UIAlertController alertControllerWithTitle:NSLocalizedString(@"Change Site", @"Title of an alert prompting the user that they are about to change the blog they are posting to.")
+                                                                             message:NSLocalizedString(@"Choosing a different site will lose edits to site specific content like media and categories. Are you sure?", @"And alert message warning the user they will loose blog specific edits like categories, and media if they change the blog being posted to.")
+                                                                      preferredStyle:UIAlertControllerStyleAlert];
+    [alertController addDefaultActionWithTitle:NSLocalizedString(@"OK",@"") handler:^(UIAlertAction *action) {
+       [self showBlogSelector]; 
+    }];
+    [alertController addCancelActionWithTitle:NSLocalizedString(@"Cancel",@"") handler:nil];
+    [self presentViewController:alertController animated:YES completion:nil];
 }
 
 - (void)showBlogSelector
 {
-    if (IS_IPAD && self.blogSelectorPopover.isPopoverVisible) {
-        [self.blogSelectorPopover dismissPopoverAnimated:YES];
-        self.blogSelectorPopover = nil;
-    }
-    
     void (^dismissHandler)() = ^(void) {
-        if (IS_IPAD) {
-            [self.blogSelectorPopover dismissPopoverAnimated:YES];
-        } else {
-            self.dismissingBlogPicker = YES;
-            [self dismissViewControllerAnimated:YES completion:nil];
-            self.dismissingBlogPicker = NO;
-        }
+        self.dismissingBlogPicker = YES;
+        [self dismissViewControllerAnimated:YES completion:nil];
+        self.dismissingBlogPicker = NO;
```

Comment:
You can kill the `self.dismissingBlogPicker` property. Just checked, it's not being used anywhere